### PR TITLE
[11.0][MIG] web_drop_target

### PR DESCRIPTION
**Pull request by grindtildeath**
TODO :
 - [ ] ~Include https://github.com/OCA/web/pull/971~ (To be done in #975 )

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]
looks good! Better drop the translation commit to have this regenerated.

You might also be interested in #971

File: web_drop_target/README.rst
Comment (simahawk) [inline web_drop_target/README.rst:1]: pls, use new readme structure

File: web_drop_target/__init__.py
Comment (simahawk) [inline web_drop_target/__init__.py]: remove coding

File: web_drop_target/static/src/js/web_drop_target.js
Comment (simahawk) [inline web_drop_target/static/src/js/web_drop_target.js]: not needed :)

**Review by simahawk — CHANGES_REQUESTED** [review]
plus, maybe we can split multi upload to another PR since is still a WIP on v10?
meanwhile we can merge this and keep next review smaller ;)

**Comment by grindtildeath** [comment]
@hbrunn @simahawk Updated according to your comments. I did open a new PR for the multiple file upload so this one can be merged when reviews are ok.

**Review by simahawk — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by inspiredbusiness** [comment]
@grindtildeath How is this PR going? What was left to see it merged in?

**Comment by simahawk** [comment]
@inspiredbusiness AFAIR it's all good and we are using it in prod since a while.
Have you tried it?

**Comment by simahawk** [comment]
BTW there's a 3rd approval not showing up because the user is gone... we can merge.
@grindtildeath can you squash your last commit?

**Comment by inspiredbusiness** [comment]
@simahawk Thanks. That's great to hear

**Comment by grindtildeath** [comment]
Sorry I missed your comments on this. Feel free to review #975 as well. :relaxed:

**Review by luc-demeyer — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Can you please squash 2 first commits and 2 last commits together before merging?

**Comment by grindtildeath** [comment]
@pedrobaeza Done !

**Pull request merged**
